components/script/dom/htmlprogresselement.rs

```diff
@@ -48,4 +54,56 @@ impl HTMLProgressElement {
 impl HTMLProgressElementMethods for HTMLProgressElement {
     // https://html.spec.whatwg.org/multipage/#dom-lfe-labels
     make_labels_getter!(Labels, labels_node_list);
+
+    // https://html.spec.whatwg.org/multipage/#attr-progress-value
```

Comment:
These doc comments should reference https://html.spec.whatwg.org/multipage/form-elements.html#dom-progress-value and https://html.spec.whatwg.org/multipage/form-elements.html#dom-progress-max instead.